content/millennium/r4/base/individuals/person.md

```diff
@@ -55,22 +61,16 @@ Notes:
 
 #### Request
 
-    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?_id=12724067
+    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?identifier=urn%3Aoid%3A2.16.840.1.113883.6.1000%7C31577
```

Comment:
Are you wanting to have the %AA codes here, or just colons and vertical pipes?  I know some systems have trouble passing the raw characters, but for readability, I think it looks better with how we had it in the DSTU2 resource with the colons displayed.

Reply:
I've seen it both ways (🙄), but I agree
```suggestion
    GET https://fhir-open.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Person?identifier=urn:oid:2.16.840.1.113883.6.1000|31577
```